.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-docstring-first
+    -   id: check-json
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.0
+    hooks:
+    -   id: flake8
+        additional_dependencies: [flake8-typing-imports==1.7.0]
+-   repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: v1.5.6
+    hooks:
+    -   id: autopep8
+-   repo: https://github.com/asottile/reorder_python_imports
+    rev: v2.4.0
+    hooks:
+    -   id: reorder-python-imports
+        language_version: python3
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v2.11.0
+    hooks:
+    -   id: pyupgrade
+        language_version: python3
```

Comment:
https://github.com/windup/windup_integration_test/blob/master/.pre-commit-config.yaml#L7 

Add check for black too

Reply:
And in the checks section I don't see the pre-commit running . 
For the pre-commit checks to run in github you need .github/workflows/main.yaml .
Ex: https://github.com/windup/windup_integration_test/blob/master/.github/workflows/main.yml

Reply:
Made the changes.
![actions-github](https://user-images.githubusercontent.com/43022982/112790578-4ea9e180-907d-11eb-9740-325d563caaec.png)